src/keybindings.h

```diff
@@ -185,7 +185,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_GOTO_TAGDEFINITION,				/**< Keybinding. */
 	GEANY_KEYS_SEARCH_NEXTMESSAGE,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOEND,			/**< Keybinding. */
-	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,			/**< Keybinding. */
+	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
```

Comment:
Aligning stuff with tabs is stupid, every time I see this I want to go in and replace it all with spaces so it looks the same and lines up everwhere :)